### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by tonglil**
Properly label a bunch of plugins with the right tags.

### CODE REVIEW

File: content/e20co/drone-marathon/index.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-author: "dellintosh"
 date: 2017-01-06T18:00:00-06:00
-image: "e20co/drone-marathon"
-repo: "e20co/drone-marathon"
+title: Marathon
+author: dellintosh
 tags: [ publish, marathon ]
+repo: expansioncap/drone-marathon
```
Comment (tonglil) [inline content/e20co/drone-marathon/index.md]: So there are actually a few repos, and I am not sure which one is most up-to-date:

https://github.com/dellintosh/drone-marathon
https://github.com/expansioncap/drone-marathon
https://github.com/drone-plugins/drone-marathon

File: content/e20co/drone-marathon/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 date: 2017-01-06T18:00:00-06:00
-image: "e20co/drone-marathon"
-repo: "e20co/drone-marathon"
-tags: [ publish, marathon ]
+title: Marathon
+author: dellintosh
+tags: [ deploy, marathon ]
 logo: marathon.svg
-title: marathon
+repo: expansioncap/drone-marathon
```
Comment (tonglil) [inline content/e20co/drone-marathon/index.md]: So there are actually a few repos, and I am not sure which one is most up-to-date:

dellintosh/drone-marathon
expansioncap/drone-marathon
drone-plugins/drone-marathon

**Comment by tonglil** [comment]
PTAL @appleboy.

File: content/e20co/drone-marathon/index.md (hunk 2 above)
Comment (tboerger) [reply content/e20co/drone-marathon/index.md]: https://github.com/expansioncap/drone-marathon/blob/master/.drone.yml this seems to be the right one.

File: content/robertstettner/drone-codecov/index.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 date: 2017-07-19T00:00:00+00:00
```
Comment (tboerger) [inline content/robertstettner/drone-codecov/index.md]: This one have been moved to drone-plugins

File: content/vallard/drone-kube/index.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
 date: 2016-01-01T00:00:00+00:00
-title: Drone-Kube
+title: Drone Kube
```
Comment (tboerger) [inline content/vallard/drone-kube/index.md]: This should be just Kube? Or Kubernetes?

File: content/robertstettner/drone-codecov/index.md (hunk 3 above)
Comment (tboerger) [reply content/robertstettner/drone-codecov/index.md]: Scratch that...

File: content/vallard/drone-kube/index.md (hunk 4 above)
Comment (tonglil) [reply content/vallard/drone-kube/index.md]: I kept it as it was read in the repo

**Pull request merged**